sensor/common/sensor/central_communication_impl.go

```diff
@@ -342,7 +342,7 @@ func (s *centralCommunicationImpl) initialConfigSync(stream central.SensorServic
 		return errors.Errorf("initial message received from Sensor was not a cluster config: %T", msg.Msg)
 	}
 	// Send the initial cluster config to the config handler
-	if err := handler.ProcessMessage(msg); err != nil {
+	if err := handler.ProcessMessage(context.TODO(), msg); err != nil {
```

Comment:
I think it would be good to propagate the context from the `initialSync` caller in the current function and pass it on to the `ProcessMessage` calls.